zkevm/tests/aggregation_tests.rs

```diff
@@ -97,9 +96,9 @@ fn verifier_circuit_verify(d: &str) {
     assert!(verifier.verify_agg_circuit_proof(agg_proof).is_ok())
 }
 
-#[cfg(feature = "prove_verify")]
+//#[cfg(feature = "prove_verify")]
 #[test]
-fn test_4in1() {
+fn test_5in1() {
```

Comment:
change it to test_agg ... so we don't need to change it to test_6in1 later ..

Reply:
good to me.

Reply:
updated in [735d170](https://github.com/scroll-tech/common-rs/pull/55/commits/735d170eb1488ec480fecaa25fa3e34093a228a0)